src/WorkerFarm.js

```diff
@@ -9,7 +9,10 @@ let shared = null;
 class WorkerFarm extends Farm {
   constructor(options) {
     let opts = {
-      maxConcurrentWorkers: getNumWorkers()
+      maxConcurrentWorkers: getNumWorkers(),
+      maxConcurrentCallsPerWorker: process.env.PARCEL_WORKER_CALLS
+        ? parseInt(process.env.PARCEL_WORKER_CALLS)
+        : 10
     };
```

Comment:
A few changes:
1. Provide a radix to `parseInt`
2. Only define `maxConcurrentCallsPerWorker` if the env variable is defined so that `worker-farm` can continue to own the default.
3. worker-farm supports `Infinity` for `maxConcurrentCallsPerWorker`, let's handle that

```js
let opts = {
  maxConcurrentWorkers: getNumWorkers()
};

if (!isNaN(process.env.PARCEL_WORKER_CALLS)) {
	opts.maxConcurrentCallsPerWorker = isFinite(process.env.PARCEL_WORKER_CALLS)
		? parseInt(process.env.PARCEL_WORKER_CALLS, 10)
		: Infinity;
}

```

Reply:
May also consider naming it with the concurrency in mind. Current name implies the work is called x times. 

`PARCEL_WORKER_CONCURRENT_CALLS`

Reply:
Done! Sorry for the late response